JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -78,8 +79,9 @@ import com.example.jetnews.ui.theme.JetnewsTheme
 private val defaultSpacerSize = 16.dp
 
 @Composable
-fun PostContent(post: Post, modifier: Modifier = Modifier) {
+fun PostContent(post: Post, state: LazyListState, modifier: Modifier = Modifier) {
```

Comment:
What about adding a default value for `state` i.e. `state: LazyListState = rememberLazyListState()`?

Would make hoisting the state optional and clean up Preview usage.

Reply:
Done